src/database/initialization/organization_to_json.sql

```diff
@@ -1,11 +1,40 @@
-CREATE OR REPLACE FUNCTION organization_to_json(organization organizations)
+-- An explicit DROP is needed to remove the old function instead of overloading it.
+DROP FUNCTION IF EXISTS organization_to_json(organizations);
+CREATE OR REPLACE FUNCTION organization_to_json(organization organizations, authenticationId VARCHAR DEFAULT NULL)
 RETURNS JSONB AS $$
+DECLARE
+  proposal_field_values_json JSONB;
 BEGIN
+  SELECT jsonb_agg(
+    proposal_field_value_to_json(pfv.*)
+    ORDER BY bf.id
+  )
+  INTO proposal_field_values_json
+  FROM proposal_field_values pfv
+  -- Remove field values for unauthenticated users while also (re)validating the user ID:
+  INNER JOIN users u
+    ON u.authentication_id = authenticationId
```

Comment:
Is this `join` too clever? At first I tried using a `case when` statement but after not getting it to work the way I wanted I thought I'd try this approach. The reason for the in-line comment is that the join is a more subtle check than a `case when`.